codex-rs/core/src/features.rs

```diff
@@ -292,4 +294,10 @@ pub const FEATURES: &[FeatureSpec] = &[
         stage: Stage::Experimental,
         default_enabled: false,
     },
+    FeatureSpec {
+        id: Feature::ShellTool,
+        key: "shell_tool",
+        stage: Stage::Stable,
+        default_enabled: true,
```

Comment:
Should we flip the sense of this so this is `default_enabled: false` like everything else? `Feature::UserProvidedShellTool` or something like that?